ui/page/settings_page.go

```diff
@@ -101,8 +102,9 @@ func NewSettingsPage(l *load.Load) *SettingsPage {
 	currencyMap[DefaultExchangeValue] = values.StrNone
 	currencyMap[components.USDExchangeValue] = values.StrUsdBittrex
 
-	currencyPreference := preference.NewListPreference(pg.WL.Wallet, pg.Theme,
-		dcrlibwallet.CurrencyConversionConfigKey, DefaultExchangeValue, currencyMap).
+	currencyPreference := preference.NewListPreference(pg.WL.Wallet, pg.Load,
+		dcrlibwallet.CurrencyConversionConfigKey, DefaultExchangeValue,
+		currencyMap, pg.Icons.ContentClear).
```

Comment:
```suggestion
	currencyPreference := preference.NewListPreference(pg.WL.Wallet, pg.Load,
		dcrlibwallet.CurrencyConversionConfigKey, DefaultExchangeValue,
		currencyMap).
```
same comment as above.